### DISCUSSION TITLE
VCA: Error (physics): Wheel shape not found for 'getAxleSpeed'

### DISCUSSION THREAD

**Discussion by RumsdeBums** [Mogli12/KeyboardSteer#749] [Previous version]
Hallo zusammen.

Habe heute diesen Fehler bemerkt.

Error (physics): Wheel shape not found for 'getAxleSpeed', object t6_axis_component2 wheelShapeIndex 37

Getestet mit der Version 1.2.0.0 und mit Standard Fahrzeuge sowie den Stoll FZ 30/60

Deaktiviere ich VCA bekomme ich diese Errormeldung nicht mehr.

**Comment by djdanby** [comment]
Selbes Problem bei mir in Version 9.1.1.3 (LS22). Der Fehler kommt offenbar nur beim Vehicle Reload. Woran liegt das denn?

**Comment by Mogli12** [comment]
Ich denke, dass beim Reload für einen oder wenige Roundtrips the I3D-Knoten nicht korrekt zugeordnet sind. Für die Verarbeitung der Differentiale verwendet VCA getAxleSpeed. 
Ich denke, dass man die Meldung ignorieren kann.
